tests/containers/buildah_integration.pm

```diff
@@ -37,6 +37,7 @@ sub run_tests {
         STORAGE_DRIVER => $storage_driver,
         BATS_TMPDIR => $tmp_dir,
         TMPDIR => $tmp_dir,
+        PATH => '/usr/local/bin:$PATH:/usr/sbin:/sbin',
```

Comment:
Why in this order? :thinking:

Reply:
We need /usr/local/bin first because we install ncat & bats on it. Then at last /usr/sbin & /sbin because these paths are not present as rootless.